mat64/dense.go

```diff
@@ -104,10 +116,16 @@ func (m *Dense) isZero() bool {
 	return m.mat.Stride == 0
 }
 
+// Dims returns the number of rows and columns in the matrix.
 func (m *Dense) Dims() (r, c int) { return m.mat.Rows, m.mat.Cols }
 
+// Caps returns the number of rows and columns in the backing matrix. This
```

Comment:
There is a trailing "This" in the comment. Not sure if there needs to be more or less.